### PR TITLE
fix: add zarf var for k3d_extra_args

### PR DESCRIPTION
**Pull request by zachariahmiller**
## Description
add zarf var to provide additional k3d config arguments 

## Related Issue

Fixes #

Relates to #

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Other (security config, docs update, etc)

## Checklist before merging

- [x] Test, docs, adr added or updated as needed
- [ ] [Contributor Guide Steps](https://github.com/defenseunicorns/uds-template-capability/blob/main/CONTRIBUTING.md)(https://github.com/defenseunicorns/uds-template-capability/blob/main/CONTRIBUTING.md#submitting-a-pull-request) followed

### CODE REVIEW

**Review by mikevanhemert — APPROVED** [review]

**Review by mjnagel — APPROVED** [review]

**Review by jeff-mccoy — COMMENTED** [review]
will the example in this readme break the RWX setup we have?

**Comment by mjnagel** [comment]
@jeff-mccoy that's a good point, it would currently break RWX without an extra arg to volume mount a common directory to the nodes for RWX storage 🤔

**Comment by zachariahmiller** [comment]
Fair, but that's only if you need RWX. Alternatively, the README example could be updated to set the debug k3s arg used in the pepr k3d setup npm script.

File: README.md [hunk 1]
```diff
@@ -32,4 +32,16 @@ This package is published via CI, but can be created locally with the following
...
 ## Additional Info
 
+You can set extra k3d args by setting the deploy-time ZARF_VAR_K3D_EXTRA_ARGS.
+Set Extra k3d args:
+
+```yaml
+package:
+  deploy:
+    set:
+      k3d_extra_args: "--agents 2"
```
Comment (jeff-mccoy) [inline README.md:42]: please update the example to something that doesn't break RWX

Comment (mjnagel) [inline README.md:42]:
```suggestion
You can set extra k3d args by setting the deploy-time `ZARF_VAR_K3D_EXTRA_ARGS`. For example to add a GPU to your cluster via a zarf config file:

```yaml
package:
  deploy:
    set:
      k3d_extra_args: "--gpus=1"
```

Comment (zachariahmiller) [reply README.md]: done

**Review by jeff-mccoy — APPROVED** [review]
Thanks!

**Pull request merged**